plugin.go

```diff
@@ -169,6 +170,23 @@ func (p Plugin) Exec() error {
 				)
 			}
 		}
+
+		// Function checks if int is in slice of ints
+		f := func(s []int, e int) bool {
+			for _, a := range s {
+				if a == e {
+					return true
+				}
+			}
+			return false
+		}
+
+		if len(p.Config.ValidCodes) > 0 && !f(p.Config.ValidCodes, resp.StatusCode) {
+			err := fmt.Errorf("Response code %d not found among valid response codes", resp.StatusCode)
+			fmt.Printf("Error: %s\n", err)
```

Comment:
No need to print it here, if you return the error it will be done within main.go